deploy/operator.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     spec:
       containers:
         - name: che-operator
-          image: quay.io/eclipse/che-operator:nightly
+          image: quay.io/eclipse/che-operator:7.12.1
```

Comment:
Why do you hardcode 7.12.1 here, but in org_v1_che_cr.yaml  you remove hard coding?